common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -1806,6 +1806,12 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
 
     // CBO related
     HIVE_CBO_ENABLED("hive.cbo.enable", true, "Flag to control enabling Cost Based Optimizations using Calcite framework."),
+    HIVE_CBO_FALLBACK_STRATEGY("hive.cbo.fallback.strategy", "CONSERVATIVE", new StringSet("NEVER, CONSERVATIVE, ALWAYS, TEST"),
```

Comment:
this seem like the stringset has only one value which is `NEVER, CONSERVATIVE, ALWAYS, TEST`

Reply:
Good catch @kgyrtkirk ! Fixed in https://github.com/apache/hive/pull/1875/commits/a0ed3355de858b9aa49ae6989966451b04029353 and added tests in https://github.com/apache/hive/pull/1875/commits/bb7c60ff43257f6a9de4db125a71e6f2328017ee and https://github.com/apache/hive/pull/1875/commits/2d28db68a1830c2c4c9212d3650cd78db5375a4e.